### DISCUSSION TITLE
Bug with Eslint plugin using node 20 .toSorted

### DISCUSSION THREAD

**Discussion by zackbcom** [zwave-js/zwave-js#6579] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
Running `yarn run lint:zwave` returns error from dev container with unsupported function.
`Array.toSorted` is not supported in node 18. [source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted#browser_compatibility)

What did you expect to happen instead?
Not have error on execution.

  1. Start dev container (node 18)
  2. Have a wrongly sorted param in a config file.
  3. execute `yarn run lint:zwave`

Error:
```bash
@zwave-js/config:lint:zwave: Oops! Something went wrong! :(
@zwave-js/config:lint:zwave: 
@zwave-js/config:lint:zwave: ESLint: 8.48.0
@zwave-js/config:lint:zwave: 
@zwave-js/config:lint:zwave: TypeError: propsWithComments.toSorted is not a function
@zwave-js/config:lint:zwave: Occurred while linting /workspaces/node-zwave-js/packages/config/config/devices/0x027a/zen37.json:64
@zwave-js/config:lint:zwave: Rule: "@zwave-js/consistent-device-config-property-order"
@zwave-js/config:lint:zwave:     at JSONProperty[key.value='paramInformation'] > JSONArrayExpression > JSONObjectExpression (/workspaces/node-zwave-js/packages/eslint-plugin/build/rules/consistent-device-config-property-order.js:58:60)
@zwave-js/config:lint:zwave:     at ruleErrorHandler (/workspaces/node-zwave-js/node_modules/.store/eslint-npm-8.48.0-0dd1c36629/node_modules/eslint/lib/linter/linter.js:1050:28)
@zwave-js/config:lint:zwave:     at /workspaces/node-zwave-js/node_modules/.store/eslint-npm-8.48.0-0dd1c36629/node_modules/eslint/lib/linter/safe-emitter.js:45:58
@zwave-js/config:lint:zwave:     at Array.forEach (<anonymous>)
@zwave-js/config:lint:zwave:     at Object.emit (/workspaces/node-zwave-js/node_modules/.store/eslint-npm-8.48.0-0dd1c36629/node_modules/eslint/lib/linter/safe-emitter.js:45:38)
@zwave-js/config:lint:zwave:     at NodeEventGenerator.applySelector (/workspaces/node-zwave-js/node_modules/.store/eslint-npm-8.48.0-0dd1c36629/node_modules/eslint/lib/linter/node-event-generator.js:297:26)
@zwave-js/config:lint:zwave:     at NodeEventGenerator.applySelectors (/workspaces/node-zwave-js/node_modules/.store/eslint-npm-8.48.0-0dd1c36629/node_modules/eslint/lib/linter/node-event-generator.js:326:22)
@zwave-js/config:lint:zwave:     at NodeEventGenerator.enterNode (/workspaces/node-zwave-js/node_modules/.store/eslint-npm-8.48.0-0dd1c36629/node_modules/eslint/lib/linter/node-event-generator.js:340:14)
@zwave-js/config:lint:zwave:     at CodePathAnalyzer.enterNode (/workspaces/node-zwave-js/node_modules/.store/eslint-npm-8.48.0-0dd1c36629/node_modules/eslint/lib/linter/code-path-analysis/code-path-analyzer.js:795:23)
@zwave-js/config:lint:zwave:     at /workspaces/node-zwave-js/node_modules/.store/eslint-npm-8.48.0-0dd1c36629/node_modules/eslint/lib/linter/linter.js:1085:32
@zwave-js/config:lint:zwave: ERROR: command finished with error: command (/workspaces/node-zwave-js/packages/config) yarn run lint:zwave exited (1)
command (/workspaces/node-zwave-js/packages/config) yarn run lint:zwave exited (1)
```

### Software versions

Driver (node-zwave-js): Master branch

### Device information

N/A

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

N/a Dev container bug

**Comment by AlCalzone** [comment, marked as answer]
Solved by https://github.com/zwave-js/node-zwave-js/pull/6580, thank you!
